components/TileItem.vue

```diff
@@ -7,12 +7,18 @@ const props = defineProps<{
 }>();
 
 const { setQueue } = useNuxtApp().$mediaPlayer;
+const origin = () => `Tile`;
```

Comment:
Is there a reason for this to be a function?

Reply:
I think Tile is the only one that doesn't have a dynamic value. So I thought it's more consistent. But it can just as well be constant.